src/components/Proposal/ProposalData.tsx

```diff
@@ -120,9 +120,9 @@ export default withSubscription({
       return combineLatest(
         proposal.state(), // state of the current proposal
         // TODO: do these subscription on the schemespage level
-        proposal.votes({where: { voter: currentAccountAddress }}, { subscribe: true }),
-        proposal.stakes({where: { staker: currentAccountAddress }}, { subscribe: true }),
-        proposal.rewards({ where: {beneficiary: currentAccountAddress}},{ subscribe: true } )
+        proposal.votes({where: { voter: currentAccountAddress }}),
+        proposal.stakes({where: { staker: currentAccountAddress }}),
+        proposal.rewards({ where: {beneficiary: currentAccountAddress}})
```

Comment:
What do these do then?  Can you please add a comment describing what descriptions are or are not being created, and if no subscription then where else is the subscription happening?

Reply:
I've created this doc that I hope makes things ab it more clear...https://github.com/daostack/alchemy/blob/01b911ba4ba4be5ec3da0138f7ab84098fdbdf10/docs/fetchingandsubscribingdata.md